python/lsst/ts/watcher/rules/cp_verify_alarm.py

```diff
@@ -0,0 +1,296 @@
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CpVerifyAlarm"]
+
+import yaml
+import json
+import lsst.daf.butler as dafButler
+import salobj
+
+from lsst.ts.idl.enums.Watcher import AlarmSeverity
+from lsst.ts import watcher
+
+
+class CpVerifyAlarm(watcher.BaseRule):
+    """Monitor state of cp_verify executions from the OCPS
+    for a particular type of calibration (BIAS, DARK, FLAT, etc).
+
+    Set alarm severity NONE if the cp_verify tests passed,
+    and issue a WARNNING  alarm if not.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"CpVerifyAlarm.{ocps_index}:{calibration_type}",
+    where ``calibration_type`` and ``ocps_index`` are derived from
+    ``config.calibration_type`` and ``config.ocps_index``, respectively.
+    """
+
+    def __init__(self, config):
+        remote_name = "OCPS"
+        remote_index = config.ocps_index
+        remote_info = watcher.RemoteInfo(
+            name=remote_name,
+            index=remote_index,
+            callback_names=["job_result"],
+            poll_names=[],
+        )
+        super().__init__(
+            config=config,
+            name=f"CpVerifyAlarm.{remote_info.index}:{remote_info.name}",
+            remote_info_list=[remote_info],
+        )
+
+        self.calibration_type = config.calibration_type
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description: Configuration for CpVerifyBiasAlarm
+            type: object
+            properties:
+                calibration_type:
+                    description: >-
+                        Alarm calibration type: BIAS, DARK, or FLAT.
+                    type: string
+                ocps_index:
+                    OCPS index (e.g., OCPS:1: LATISS; OCPS:2: LSSTComCam).
+                verification_threshold:
+                    type: integer
+                    descriptor: Maximum number of failures per detector per test type.
+                default: 8
+            required: [calibration_type, ocps_index]
```

Comment:
Please include `verification_threshold`. It will always be present in any case, since it has a default.